lib/fakes3/server.rb

```diff
@@ -378,9 +378,13 @@ def normalize_delete(webrick_req, s_req)
         end
 
         if elems.size == 0
-          s_req.type = Request::DELETE_OBJECTS
-          s_req.query = query
-          s_req.webrick_request = webrick_req
+          if !s_req.is_path_style
```

Comment:
How about just a positive conditional rather than a negation for easier reading.